examples/worldview/src/features/map/mapSlice.js

```diff
@@ -12,9 +11,6 @@ const mapSlice = createSlice({
   reducers: {
     updateViewState: (state, action) =>
       void (state.viewState = {...state.viewState, ...action.payload})
-  },
-  extraReducers: builder => {
-    builder.addCase('@@kepler.gl/REGISTER_ENTRY', state => void (state.ready = true));
```

Comment:
Moved out of mapSlice to appSlice, since mapSlice will soon be moved into the lib and we don't want to bring in unnecessary dependencies on kepler logic.